project/ProjectPlugin.scala

```diff
@@ -223,6 +223,16 @@ object ProjectPlugin extends AutoPlugin {
       includeFilter in Jekyll := "*.html" | "*.css" | "*.png" | "*.jpg" | "*.gif" | "*.js" | "*.swf" | "*.md"
     )
 
+    lazy val legacyAvroDecimalProtocolSettings: Seq[Def.Setting[_]] = Seq(
+      publishMavenStyle := true,
+      crossPaths := false,
+      libraryDependencies := Nil
+    )
+
+    lazy val legacyAvroDecimalModelSettings: Seq[Def.Setting[_]] = Nil
+
+    lazy val legacyAvroDecimalEncodersSettings: Seq[Def.Setting[_]] = Nil
```

Comment:
We have the Scala model and the avdl type defined with the same name and the same package. In this way, the encoders can depend on the scala model for creating the serializers and the clients using avdl need to write something like this:
https://github.com/frees-io/freestyle-rpc-protocol-decimal-update/blob/master/server-v2/build.sbt#L25

Do someone knows how to exclude a "`dependsOn`" module dependency from the dependencies of that module in the pom?